mixin/panels.libsonnet

```diff
@@ -0,0 +1,766 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+
+{
+  new(this)::
+    {
+      local signals = this.signals,
+
+      // Overview Dashboard Panels - Row 1 Statistics
+      yesterdayCostStat:
+        commonlib.panels.generic.stat.base.new(
+          'Total cost (24h window)',
+          targets=[signals.overview.yesterdayCost.asTarget()],
+          description='Total cost estimate for the past 24 hours. This is a rolling 1-day window from exporter.'
+        )
+        + g.panel.stat.options.withGraphMode('none')
+        + g.panel.stat.options.withTextMode('value')
+        + g.panel.stat.standardOptions.withUnit('currencyUSD')
+        + g.panel.stat.standardOptions.withDecimals(2),
+
+      dodCostDeltaStat:
+        commonlib.panels.generic.stat.base.new(
+          'Cost change % (24h vs 24h ago)',
+          targets=[signals.overview.dodCostDelta.asTarget()],
+          description='Percentage change comparing current 24h window to the 24h window from yesterday. Green = low growth, Yellow = moderate growth (>15%), Red = high growth (>25%).'
+        )
+        + g.panel.stat.options.withGraphMode('none')
+        + g.panel.stat.options.withTextMode('value')
+        + g.panel.stat.standardOptions.withUnit('percentunit')
+        + g.panel.stat.standardOptions.withDecimals(1)
+        + g.panel.stat.standardOptions.thresholds.withSteps([
+          g.panel.stat.thresholdStep.withColor('green'),
+          g.panel.stat.thresholdStep.withColor('yellow') + g.panel.stat.thresholdStep.withValue(0.15),
+          g.panel.stat.thresholdStep.withColor('red') + g.panel.stat.thresholdStep.withValue(0.25),
+        ]),
+
+      totalDbusConsumedStat:
+        commonlib.panels.generic.stat.base.new(
+          'Total DBUs (24h window)',
+          targets=[signals.overview.totalDbusConsumed.asTarget()],
+          description='Total DBUs consumed over the past 24 hours. This is a rolling 1-day window from the exporter.'
+        )
+        + g.panel.stat.options.withGraphMode('none')
+        + g.panel.stat.options.withTextMode('value')
+        + g.panel.stat.standardOptions.withUnit('none'),  // would be 'dbu' but no custom unit available
```

Comment:
You could create a custom unit? https://github.com/grafana/grafonnet/blob/8a27651b56fbdba45a389ccb11440200091c73a1/docs/API/panel/stat/index.md#fn-standardoptionswithunit

Reply:
This shouldn't have been resolved? I don't see any changes to the mixin? 
<img width="202" height="83" alt="image" src="https://github.com/user-attachments/assets/1158b248-0aeb-4141-bb7f-f6099dd07e9a" />